composer.json

```diff
@@ -3,13 +3,13 @@
     "description": "Template file parsing library",
     "type": "library",
     "require": {
-        "php": "~7.1",
+        "php": "^8.1",
         "symfony/console": "^2.8|^3.2|^4.0|^5.0"
```

Comment:
Can you change that to `^5.4|^6.0`? So we can remove non supported versions as well.

Reply:
Nevermind, will do it in master. As I remove travis either way :)